packages/typespec-ts/test/unit/parametersGenerator.spec.ts

```diff
@@ -5,27 +5,6 @@ import { assertEqualContent } from "../util/testUtil.js";
 describe("Parameters.ts", () => {
   describe("query parameters", () => {
     describe("apiVersion in query", () => {
-      it("should't generate apiVersion if there's a client level apiVersion", async () => {
```

Comment:
why remove this?

Reply:
just because it is duplicated with UTs in apiVersion.spec.ts